EIPS/eip-6188.md

```diff
@@ -0,0 +1,44 @@
+---
+eip: 6188
+title: Nonce Cap
+description: Caps the nonce at 2^64-2
+author: Pandapip1 (@Pandapip1)
+discussions-to: https://ethereum-magicians.org/t/eip-6190-functional-selfdestruct/12232
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-12-20
+requires: 2929
+---
+
+## Abstract
+
+This EIP reserves a nonce for special contracts. It also caps the nonce at `2^64-2`.
```

Comment:
What is a special contract? Based on the title, it sounds like the primary purpose of the EIP is to cap the nonce (so using "It also" seems weird.)

Reply:
I'll rephrase for clarity.